Kitodo/src/main/webapp/WEB-INF/templates/includes/metadataEditor/dialogs/errorLoadingMetadataDialog.xhtml

```diff
@@ -0,0 +1,44 @@
+<!--
+ *
+ * (c) Kitodo. Key to digital objects e. V. <contact@kitodo.org>
+ *
+ * This file is part of the Kitodo project.
+ *
+ * It is licensed under GNU General Public License version 3 or later.
+ *
+ * For the full copyright and license information, please read the
+ * GPL3-License.txt file that was distributed with this source code.
+ *
+-->
+
+<ui:composition
+        xmlns="http://www.w3.org/1999/xhtml"
+        xmlns:h="http://xmlns.jcp.org/jsf/html"
+        xmlns:ui="http://xmlns.jcp.org/jsf/facelets"
+        xmlns:p="http://primefaces.org/ui">
+    <p:dialog id="errorLoadingMetadataDialog"
+              widgetVar="errorLoadingMetadataDialog"
+              width="500px"
+              showHeader="false"
+              visible="#{not empty DataEditorForm.metadataFileLoadingError}"
+              appendTo="@(body)"
+              resizable="false"
+              closable="false"
```

Comment:
Looks good! 
I would just like to add this line of code in order to hide the loadingScreen displayed behind the dialog box.
` onShow="if($('#loadingScreen').length) {$('#loadingScreen').hide();}"`